stable/node-local-dns/templates/servicemonitor.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   name: {{ include "node-local-dns.fullname" . }}
-  namespace: kube-system
+  namespace: {{ .Values.serviceMonitor.prometheusNamespace}}
```

Comment:
Is it typical to create this `ServiceMonitor` resource in a different namespace to the rest of the resources? If not, then I would say we could just make a single value for the namespace of all resources? Or alternatively remove `namespace` from all resources and rely on `--namespace` in Helm?

Reply:
OK, makes sense, thanks